to_done_to_html.py

```diff
@@ -0,0 +1,196 @@
+"""
+Simple update to the ToDone plugin to export ToDos to HTML. This is useful if you want to share
+your TODOs via email.
+
+This script by Anjul Patney (Original plugin here: https://github.com/tiffon/sublime-to-done)
+"""
+import html
+import re
+import tempfile
+import webbrowser
+import sublime
+import sublime_plugin
+
+# Regex taken from https://gist.github.com/gruber/249502
+URL_REGEX = r'(?i)\b((?:[a-z][\w-]+:(?:/{1,3}|[a-z0-9%])|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:\'".,<>?«»“”‘’]))'
+
+class ToDoneToHtmlCommand(sublime_plugin.TextCommand):
+    """ToDone to HTML Class"""
+
+    def __init__(self, view):
+        super().__init__(view)
+        self.indent_level_map = {0 : 0}
+
+    def is_visible(self):
+        """Return true if syntax is ToDone"""
+        return "ToDone" in self.view.settings().get("syntax")
+
+    def run(self, edit):
+        """Main plugin code. Collect the current selection(s) and export to HTML"""
+
+        del edit
+
+        lines = self.get_selected_text().splitlines()
+
+        processed_lines = []
+
+        last_list_level = 0
+        list_level = 0
+
+        self.parse_indents(lines)
+
+        for line in lines:
+            if line.strip() == "":
+                prefix = "<br>"
+                postfix = ""
+            elif line.strip().startswith("-") or line.strip().startswith("+"):
```

Comment:
Should this also handle task lines that start with `!` and `.`, e.g

```
!-5	this has a black bg
.-	(this was ignored)do things
```

Reply:
Easy to add! 👍